forecast-admin/forecast/opportunities/admin.py

```diff
@@ -1,12 +1,26 @@
 from django.contrib import admin
 
 # Register your models here.
-from .models import Opportunity, Office
+from .models import Opportunity, Office, OSBUAdvisor
 
 
 @admin.register(Opportunity)
 class OpportunityAdmin(admin.ModelAdmin):
 
+    def make_published(self, request, queryset):
```

Comment:
I'm sure there are complicated methods, a simple one would be: 

Create a function (outside of the class) that just takes in a query set, and marks everything in the queryset as published. That should be easy to test.  This method doesn't need the request, or self.